### PR TITLE
Smash heatmap - Heat Map gray scale website

### PR DESCRIPTION
**Pull request by carloso0114**
This website should contain the following requirements:
- Align all elements with float and flex or grid.
- Use article tag for all articles listed on the page.
- Make sure to build your layout using a width of 1440px on the original page.

### CODE REVIEW

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello @carloso0114  :wave:

__You've done an excellent job here.__

- All linter checks are green :green_circle: 
- Descriptive title and summary on your PR page :+1: 
- Well implemented README file :100: 

_However, there are some changes that must be made before this project can be approved. Please check my comments below and fix them as appropriate._

**[ REQUIRED CHANGES ]**

_This project consists of building a heatmap of the  Smashing Magazine website. Being heatmap means that **all the elements and element's backgrounds and font colors should be grayscale color (black, white, or any shade of gray)**, this rule also applies to images if you decide to add any._

- [ ] You have non-greyscale colors in your project. Please re-factor your entire page to and make sure that you only use grayscale colors.



- [ ] You have a lot of articles on your page. You should use the `<article>` tag for all the articles listed on the page.




**[ COMMENTS ]**

- It is a good practice to always move put all your image assets in a single folder, inside the `assets` directory. This makes your assets well structured and neat.

**Please make the required changes and submit another review request when you are done.**

* * *

**_You can reach out to me on twitter (@ola_lere), Github(@teekaytech), or slack (@taofeekolalere )_**
__Happy Coding :smiley:__

**Review by ghost — APPROVED** [review]
# Status: Approved

**Congratulations team and good luck on your next project 😃.**

**Pull request merged**
